app/assets/javascripts/share.js

```diff
@@ -42,29 +42,48 @@ function pageLoad() {
     }
 
     // data picker
-    if ($('.datetimepicker')) {
-        $('.datetimepicker').datetimepicker({
-            format: 'YYYY/MM/DD HH:mm',
-            stepping: 15
-        });
-        $('.datetimepicker_deadline').datetimepicker({
-            format: 'YYYY/MM/DD'
-        });
-    };
+
+    $( "#but" ).click(function() {
+      $( "#save" ).show();
+      $( "#but" ).hide();
```

Comment:
If you use some object twice or more then put it in variable to optimize app performance